lib/azure/armrest/model/base_model.rb

```diff
@@ -189,7 +190,9 @@ class ResourceProvider < BaseModel; end
     class Sku < BaseModel; end
     class Usage < BaseModel; end
 
-    class ResponseHeaders < BaseModel; end
+    class ResponseHeaders < BaseModel
+      undef_method :response_headers
```

Comment:
@bzwei Well, I need to be able to create a model object from the response.headers output from RestClient. What do you suggest then?